package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html

```diff
@@ -64,6 +64,19 @@
 		return v and translate('enabled') or translate('disabled')
 	end
 
+	local function formatBits(v, factor)
+		--to format a value given in kilobits use a factor of 1000
+		factor = factor or 1
+		units = {[0]='', 'k', 'M', 'G'}
+
+		bits = math.max(0, v*factor)
+		pow = math.floor(math.log(bits)/math.log(1000))
```

Comment:
```suggestion
		pow = math.floor(math.log(math.max(math.abs(bits),1))/math.log(1000))
```

The log() function will become negative for input values smaller than 1. This change should fix this.

Furthemore, this change should enable us to handle negative input values correctly.

Reply:
7618dcb07a3aa0c780d5cf975f4f3d26c8aa1b96